context.go

```diff
@@ -110,6 +110,14 @@ func NewContextForRGBA(im *image.RGBA) *Context {
 	}
 }
 
+// GetCurrentPoint will return the current point and if there is a current point
+func (dc *Context) GetCurrentPoint() (Point, bool) {
+	if dc.hasCurrent {
+		return Point{X: dc.current.X, Y: dc.current.Y}, true
```

Comment:
Why not just `return dc.current, true`?

Might be worth noting in the comment that the point will have undergone transformation (from the context's transformation matrix).